bundles/binding/org.openhab.binding.denon/META-INF/MANIFEST.MF

```diff
@@ -22,7 +22,6 @@ Import-Package: javax.xml.bind,
  org.apache.commons.io,
  org.apache.commons.lang,
  org.apache.commons.net,
```

Comment:
Eclipse complains that "No available bundle exports package 'org.apache.commons.net'"

Reply:
Since the `org.apache.commons.net.telnet` import was removed the general `.net` import was no longer necessary so I've also removed it.